css/main.css

```diff
@@ -13,9 +13,28 @@ body {
 html, body {
     position: relative;
     overflow-x: hidden;
+    text-align: center;
 }
 * {
     -webkit-box-sizing: border-box;
     -moz-box-sizing: border-box;
     box-sizing: border-box;
 }
+li {
+    list-style: none;
+}
+.hideItem {
+    display: none;
+}
+.bodyContainer {
+    width: 35%;
+    margin: 0 auto;
+}
+.inputBtnWrap, .prefContainer, .drinkContainer, #moreBtn {
```

Comment:
Consider refactoring this into a single class, and adding it to the elements. It will parse faster